docs/components/Permalink.tsx

```diff
@@ -73,6 +74,33 @@ const PermalinkBase = ({ component, children, className, ...rest }: BaseProps) =
     children
   );
 
+// @ts-ignore Jest ESM issue https://github.com/facebook/jest/issues/9430
+const { default: testTippy } = tippy;
+
+const PermalinkCopyIcon = ({ slug }: { slug: string }) => {
+  const tippyFunc = testTippy || tippy;
+  React.useEffect(() => {
+    tippyFunc('#docs-anchor-permalink-' + slug, {
+      content: 'Click to copy anchor link',
```

Comment:
```suggestion
      content: 'Click to copy anchor link',
      arrow: roundArrow,
      offset: [0, 0],
```

Small tweaks for the Tooltip factory.